ipa-core/src/protocol/ipa_prf/malicious_security/verifier.rs

```diff
@@ -104,32 +99,24 @@ pub fn compute_final_sum_share<F: PrimeField, const λ: usize, const P: usize>(z
 ///
 /// The function uses streams since stream offers a chunk method.
 fn recurse_u_or_v<'a, F: PrimeField, J, const λ: usize>(
-    u_or_v_stream: J,
+    u_or_v_iterator: J,
```

Comment:
it is better not to encode the type of this parameter inside the name. `u_or_v` is fine imo